src/components/layout/footer/Footer.tsx

```diff
@@ -5,7 +5,10 @@ function Footer() {
     <FooterWrapper>
       <ContentWrapper>
         <LogoText>
-          Vytvořila <a href='https://sunovy.utb.cz/'>Studentská Unie UTB</a>
+          Vytvořila{' '}
```

Comment:
ayy, could we remove the empty string behind the 'Vytvořila' word? probably a forgotten one

Reply:
it is there on purpose - it ensures the space between simple text "Vytvořila" and the link "Studentská Unie UTB"

Reply:
yea, cool. also, the &nbsp; could be used for that, but I think it shouldn't be a problem :)